docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+services:
+  web:
+    build:
+      context: .
+      target: dev
+    init: true
+    command: django-admin runserver 0.0.0.0:8000
```

Comment:
Suggestion: Perhaps make this a never-ending no-op like `tail -f /dev/null` so people can start the dev server themselves? This makes things like connecting a debugger much easier.

Reply:
@kaedroho this suggestion from Ben seems worth doing